cmd/kops/update_cluster.go

```diff
@@ -172,6 +175,20 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		glog.Infof("Using SSH public key: %v\n", c.SSHPublicKey)
 	}
 
+	var phase cloudup.Phase
```

Comment:
Let's get this in and look at the broader role of create vs update as part of the kops-server UX pass